5calls/app/src/main/java/org/a5calls/android/a5calls/FiveCallsApplication.java

```diff
@@ -19,6 +19,7 @@
 import android.app.Activity;
 import android.app.Application;
 import android.os.Bundle;
+import android.util.Log;
```

Comment:
consider removing unused import.
i realize we have a mess of unused imports across the app, though, so no worries. we should just go through and clean up at some point.